MdePkg/Test/Include/MockPeiServicesLib.h

```diff
@@ -0,0 +1,17 @@
+/** @file
+  Mock implementation of PeiServicesLib
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef MOCK_PEI_SERVICES_LIB_H_
+#define MOCK_PEI_SERVICES_LIB_H_
+
+#include <Uefi.h>
+
+typedef struct _PPI_STATUS {
+  VOID          *Ppi;
+  EFI_STATUS    Status;
+} PPI_STATUS;

```

Comment:
I wrote this, so should have been on me, but can we add a comment above this struct stating the purpose? I.e. something like:

```
Structure used to return a status and ppi pointer to a calling unit test function for mocking purposes.
```